js/app.js

```diff
@@ -37,56 +54,49 @@ Question.prototype = {
     }
 };
 
-// Details for each question
-var typeface = new Array(); //GARAMOND
-typeface[0] = new Font(
-    ".garamond",
-    ".garamond",
-    ["Verdana", "Garamond", "Monotype"],
-    "Garamond"
-);
-console.log(typeface);
-
-typeface[1] = new Font(  //HELVETICA
-    ".helvetica",
-    ".helvetica",
-    ["Baskerville", "Sans-serif", "Helvetica"],
-    "Helvetica"
-);
-console.log(typeface);
-
-typeface[2] = new Font( //TIMES
-    ".times",
-    ".times",
-    ["Goudy Old Style", "Serifa", "Times"],
-    "Times"
-);
-console.log(typeface);
-
-typeface[3] = new Font( //VERDANA
-    ".verdana",
-    ".verdana",
-    ["Verdana", "Verdy", "Script"],
-    "Verdana"
-);
-console.log(typeface);
-
-typeface[4] = new Font( //ARIAL
-    ".arial",
-    ".arial",
-    ["Comic Sans", "Trade Gothic", "Arial"],
-    "Arial"
-);
-console.log(typeface);
```

Comment:
Good first try... Some first glance things:
getTotalScore and getScore don't have the var keyword in front of them. This means they become global variables... To understand this, you have to understand the concept of scope, which generally means that the variable is only accessible inside of a function.  If you declare a variable without var, it becomes accessible outside of the function, and in most cases this is bad (I can elaborate on this next session, or if you'd like you can google this stuff).

for this line...
```javascript
 if (Question.answer == choice) {
```
this is telling me that the concept of a `Class` vs `Object` is not clear to you.  The class is kind of like a blueprint or map of a house, whereas an `object` is the actual house.  So back to our Question stuff, Question is the blueprint or description, and when we do `new Question` we create an actual question.  In order to get the actual answer of an actual question, we can't use the Question class, we have to use the object.  Hope that clears some things up...

Next, make sure you understand what these lines do
```javascript
getTotalCorrect = $('.questionCount').text(getTotalCorrect+1); 
getScore = $('.scorePercentage').text(getScore);
```
I would suggest debugging like we did in our session, and inspecting the value of your variables.  Also make sure you look up the jquery text method to understand what it is doing...

Hope this helps!